schedule/security/alp/check_tpm2.yaml

```diff
@@ -1,9 +0,0 @@
-name: check_tpm2
-description:    >
-    This is for the check_tpm2 test.
-schedule:
-    - jeos/firstrun
-    - transactional/host_config
-    - transactional/enable_selinux
-    - security/tpm2/tpm2_measured_boot
-    - security/tpm2/tpm2_fail_key_unsealing
```

Comment:
seems that you removed some not intended test coverage here @paolostivanin

Reply:
That was intended! The new test is testing more stuff. Moreover, the purpose of this simple test is to check whether or not TPM is present and working, so basically any command would do 😊

Reply:
I don't see running anywhere the test I created `tpm2_fail_key_unsealing`. Could you please check to not loose it?
As I mentioned before, perhaps it is a good idea the group all tpm2 tests in single test suite.

Reply:
I see what you mean. Yes, we should definitely schedule your test.
I don't  think we should schedule it here because the purpose of this very simple test is to just check for tpm presence (https://progress.opensuse.org/issues/134633). Ideally, it should be something quick and fast to execute because we care only about one thing in this module: checking that tpm is present and working.
For all other TPM stuff, we should think about grouping them all together, yes!

Reply:
Hi @paolostivanin after discussion offline seems that we agree to have all these test in same test suite.
Did you file a ticket for it? (it would be useful to show how tpm fails for next builds recovering this test module).
Or you prefer to postpone the refactor at least we should revert the fact that the test is not used anymore.